crates/fuel-core/src/graphql_api/storage/assets.rs

```diff
@@ -0,0 +1,48 @@
+use fuel_core_storage::{
+    blueprint::plain::Plain,
+    codec::{
+        postcard::Postcard,
+        raw::Raw,
+    },
+    structured_storage::TableWithBlueprint,
+    Mappable,
+};
+use fuel_core_types::{
+    fuel_merkle::common::Bytes32,
+    fuel_tx::{
+        AssetId,
+        ContractId,
+    },
+};
+
+/// Contract info
+pub struct AssetsInfo;
+
+pub type AssetDetails = (ContractId, Bytes32, u64); // (contract_id, sub_id, total_amount)
```

Comment:
Can we use struct with named fields? Something similar to what we have [here](https://github.com/FuelLabs/fuel-core/pull/2445/files#diff-ac9c0df77a716f8ffa16e831870ec184fd1e5066adc11825d415bcfa776cdd4bR39).